packages/strapi-admin/services/user.js

```diff
@@ -32,6 +32,16 @@ const create = async attributes => {
  * @returns {Promise<user>}
  */
 const update = async (params, attributes) => {
+  // hash password if a new one is sent
+  if (_.has(attributes, 'password')) {
+    const hashedPassword = await strapi.admin.services.auth.hashPassword(attributes.password);
+
+    return strapi.query('user', 'admin').update(params, {
+      ...attributes,
+      password: hashedPassword,
+    });
```

Comment:
Just by curiosity, why duplicating the query instead of re-assigning the password field in params?

Reply:
To avoid side effects in case the attributes passed in are used in the caller function for something else.

Reply:
Makes sense 👍